bigbluebutton-html5/imports/ui/components/user-avatar/styles.scss

```diff
@@ -49,7 +50,21 @@ $user-list-bg: $color-off-white;
 }
 
 .talking {
-  box-shadow: 0 0 0 1px $user-list-bg, 0 0 0 6px;
+  
+  animation: pulse 1.2s infinite ease-in-out;
```

Comment:
In other places in the code we have `animation` combined with `webkit-animation`. Are you sure we don't need to add it here too?